src/datashare/azext_datashare/generated/_params.py

```diff
@@ -0,0 +1,307 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
+# pylint: disable=too-many-lines
+# pylint: disable=too-many-statements
+
+from azure.cli.core.commands.parameters import (
+    tags_type,
+    get_enum_type,
+    resource_group_name_type,
+    get_location_type
+)
+from azext_datashare.action import AddIdentity
+
+
+def load_arguments(self, _):
+
+    with self.argument_context('datashare account list') as c:
+        c.argument('resource_group_name', resource_group_name_type, help='The resource group name.')
+        c.argument('skip_token', help='Continuation token')
+
+    with self.argument_context('datashare account show') as c:
+        c.argument('resource_group_name', resource_group_name_type, help='The resource group name.')
+        c.argument('account_name', help='The name of the share account.')
+
+    with self.argument_context('datashare account create') as c:
+        c.argument('resource_group_name', resource_group_name_type, help='The resource group name.')
+        c.argument('account_name', help='The name of the share account.')
+        c.argument('location', arg_type=get_location_type(self.cli_ctx), help='Location of the azure resource.')
+        c.argument('tags', tags_type, help='Tags on the azure resource.')
+        c.argument('identity', action=AddIdentity, nargs='+', help='Identity of resource')
+
+    with self.argument_context('datashare account update') as c:
+        c.argument('resource_group_name', resource_group_name_type, help='The resource group name.')
+        c.argument('account_name', help='The name of the share account.')
+        c.argument('tags', tags_type, help='Tags on the azure resource.')
+
+    with self.argument_context('datashare account delete') as c:
+        c.argument('resource_group_name', resource_group_name_type, help='The resource group name.')
+        c.argument('account_name', help='The name of the share account.')
+
+    with self.argument_context('datashare consumer-invitation list') as c:
+        c.argument('skip_token', help='Continuation token')
+
+    with self.argument_context('datashare consumer-invitation show') as c:
+        c.argument('location', arg_type=get_location_type(self.cli_ctx), help='Location of the invitation')
+        c.argument('invitation_id', help='An invitation id')
+
+    with self.argument_context('datashare consumer-invitation reject-invitation') as c:

```

Comment:
sorry if i missed this in help, but seems not in help

Reply:
command renamed to `datashare consumer invitation reject`